edx_notifications/server/api/consumer.py

```diff
@@ -154,10 +155,39 @@ def get(self, request):
             filters=filters,
             options=options,
         )
-
         resultset = [user_msg.get_fields() for user_msg in user_msgs]
+        response_data = {
+            'notifications': resultset,
+            'unread_count': sum([not notification['read'] for notification in resultset])
+        }
+        return Response(response_data, status.HTTP_200_OK)
+
+    def post(self, request):
+        """
+        HTTP POST Handler
+        """
+        notification_data = request.data.get('notification') or request.data.get('message')
```

Comment:
why we are accommodating both `notification` and `message` as data param instead of sticking to just one?